src/app/search/search-tool-table/search-tool-table.component.ts

```diff
@@ -34,6 +35,11 @@ export class SearchToolTableComponent extends SearchEntryTable implements OnInit
         this.dataSource.data = entries;
       }
     });
+    this.dataSource.sortData = (data: DockstoreTool[], sort: MatSort) => {
```

Comment:
Can you just move this to search-entry-table.ts, and then you don't have to repeat it here and in search-workflow-table.component.ts? Sorry I didn't notice before.

Reply:
ah good point